File: Iniciantes/b1019.java
import java.util.Scanner;

public class b1019 {
	public static void main(String[] args) throws Exception {
		Scanner sc = new Scanner(System.in);

		int duracaoSegundos = sc.nextInt();
		int horas = duracaoSegundos / 3600;
		duracaoSegundos -= horas * 3600;
		
		int minutos = duracaoSegundos / 60;
		duracaoSegundos -= minutos * 60;

		int segundos = duracaoSegundos;

		System.out.println(horas + ":" + minutos + ":" + segundos);	
	}
}

File: Iniciantes/b1019.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let duracaoSegundos = parseInt(lines.shift());
let horas = Math.floor(duracaoSegundos / 3600)
let minutos = Math.floor((duracaoSegundos%3600)/60)
let segundo = Math.floor((duracaoSegundos%3600) % 60)
console.log(`${horas}:${minutos}:${segundo}`)
